CHANGELOG.md

```diff
@@ -2,11 +2,12 @@
 
 Certbot adheres to [Semantic Versioning](https://semver.org/).
 
-## 0.36.0 - master
+## 0.36.0
```

Comment:
We want to keep `master` here as 0.36.0 hasn't been released yet.

Reply:
Thought I had that. Looks like I forgot.